src/ProductConstructionService/ProductConstructionService.Client/ProductConstructionServiceApiOptions.cs

```diff
@@ -2,11 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.Maestro.Common;
 
 namespace ProductConstructionService.Client
 {
     public partial class ProductConstructionServiceApiOptions
     {
+        public const string StagingPcsBaseUri = "https://product-construction-int.delightfuldune-c0f01ab0.westus2.azurecontainerapps.io/";
```

Comment:
pls add a note to the devguide in the readme that this also needs to change during a redeployment. Maybe we could even set it as an env variable during the CA creation in the bicep in the future?